React/Fabric/Mounting/ComponentViews/ProgressView/RCTProgressViewComponentView.h

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import <UIKit/UIKit.h>
+
+#import <React/RCTViewComponentView.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * UIView class for root <ProgressView> component.
```

Comment:
--root--

Reply:
@shergin all the components are like this lol
```
/**
 * UIView class for root <Slider> component.
 */
```